### PR TITLE
Arrays

### PR DESCRIPTION
**Pull request by antonredko1993**

### CODE REVIEW

File: ElementaryTasks/Arrays.cs [hunk 1]
```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace ElementaryTasks
+{
+    class Arrays
+    {
+        public void MinElement()
```
Comment (Akimmax) [inline ElementaryTasks/Arrays.cs:9]: Please use verbs or verb phrases for method name https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-type-members

File: ElementaryTasks/Program.cs [hunk 2]
```diff
@@ -86,7 +86,18 @@ static void Main(string[] args)
 
             //12) 	Вывести число, которое является зеркальным отображением последовательности цифр заданного числа
             //      например, задано число 123, вывести 321.    
-            loops.MirrorNumber();
+            //loops.MirrorNumber();
+
+            Arrays arrays = new Arrays();
+
+            //1)	Найти минимальный элемент массива
```
Comment (Akimmax) [inline ElementaryTasks/Program.cs:93]: Please use English for comment

File: ElementaryTasks/Arrays.cs [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+        {
```
Comment (GorbunowArtem) [inline ElementaryTasks/Arrays.cs:10]: Please create method which solves this problem with LINQ

File: ElementaryTasks/Arrays.cs [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+                    maxValue = numbers[i];
+                    index = i;
+                }
+            }
+            Console.WriteLine(index);
+        }
+
+        public void SumOfElements()
+        {
+            int[] numbers = { 11, 10, 12, 13, 14, 4, 16, 21, 15, 17 };
```
Comment (GorbunowArtem) [inline ElementaryTasks/Arrays.cs:41]: Please create method which solves this problem with LINQ

File: ElementaryTasks/Arrays.cs [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+                    minValue = numbers[i];
+                }
+            }
+            Console.WriteLine(minValue);
+        }
+
+        public void IndexOfMaxValue()
+        {
+            int[] numbers = { 11, 10, 12, 13, 14, 4, 16, 21, 15, 17 };
+            int maxValue = numbers[0];
```
Comment (GorbunowArtem) [inline ElementaryTasks/Arrays.cs:26]: Please create method which solves this problem with LINQ

**Pull request merged**